hack/verify-dockerless.sh

```diff
@@ -0,0 +1,15 @@
+not_have_dockerless=$(find ../pkg/kubelet/dockershim -name "*.go" ! -exec grep -q 'dockerless' {} \; -print)
+if [[ $not_have_dockerless ]]; then
+        echo "Following files do not contain dockerless tag"
+        echo $not_have_dockerless
+fi
+imports_docker=$(find ../{cmd,pkg}/kubelet/ -name "*.go" -exec grep -q 'github.com/docker/docker' {} \; -print | grep -v ../pkg/kubelet/dockershim)
+if [[ $imports_docker ]]; then
+        echo "Following files imports docker outside dockershim"
+        echo $imports_docker
```

Comment:
Similarly we want to exit w/ failure condition if any files are found.

Reply:
sure